packages/data/src/store/index.js

```diff
@@ -16,8 +16,8 @@ function createCoreDataStore( registry ) {
 				'hasFinishedResolution',
 				'isResolving',
 				'getCachedResolvers',
-			].reduce( ( memo, selectorName ) => ( {
-				...memo,
+			].reduce( ( memoAccumulator, selectorName ) => ( {
```

Comment:
A similar note here. It feels like `memoAccumulator` could be shortened to `accumulator` and it will read well enough.